plugins/captions-selector/front.js

```diff
@@ -0,0 +1,60 @@
+const { ElementFromFile, templatePath } = require("../utils");
+const { ipcRenderer } = require("electron");
+
+function $(selector) { return document.querySelector(selector); }
```

Comment:
Note for later (no change needed): this function is used in multiple plugins and can probably be refactored